startup/10-motors.py

```diff
@@ -0,0 +1,76 @@
+from ophyd import Device, Component as Cpt, EpicsMotor
+
+
+# Temporary test motor
+class TestMotor(Device):
+    e   = Cpt(EpicsMotor, '-Ax:E}Mtr')
+    f   = Cpt(EpicsMotor, '-Ax:F}Mtr')
+    ctr = Cpt(EpicsMotor, '-Ax:XCtr}Mtr')
+    gap = Cpt(EpicsMotor, '-Ax:XGap}Mtr')
+
+
+class XYMotor(Device):
+    x = Cpt(EpicsMotor, '-Ax:X}Mtr')
+    y = Cpt(EpicsMotor, '-Ax:Y}Mtr')
+
+
+class Slits(Device):
+    b     = Cpt(EpicsMotor, '-Ax:B}Mtr')
+    i     = Cpt(EpicsMotor, '-Ax:I}Mtr')
+    o     = Cpt(EpicsMotor, '-Ax:O}Mtr')
+    t     = Cpt(EpicsMotor, '-Ax:T}Mtr')
+    x_ctr = Cpt(EpicsMotor, '-Ax:XCtr}Mtr')
+    x_gap = Cpt(EpicsMotor, '-Ax:XGap}Mtr')
+    y_ctr = Cpt(EpicsMotor, '-Ax:YCtr}Mtr')
+    y_gap = Cpt(EpicsMotor, '-Ax:YGap}Mtr')
+
+
+class DCM(Device):
+    # Virtual Motor
+    energy  = Cpt(EpicsMotor, '-Ax:E}Mtr')
```

Comment:
Should it then be `Cpt(Signal, ...)`, or is it backed by a set of signals pretending to be an EPICS motor?

Reply:
This is an actual motor record, of the 'Soft Motor' DTYP

Reply:
Thanks, sounds good.